PowerUp/app/src/main/res/layout-land/completed_scene.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/background" >
+
+    <TextView
+        android:id="@+id/completed_scene_textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:ems="10"
+        android:text="@string/completed_scene"
+        android:textColor="@android:color/black"
+        android:textSize="@dimen/game_text_size"
+        android:layout_above="@+id/ContinueButtonMap"
```

Comment:
Only use `@+id` when declaring a new id. For references use `@id` only. Read the coding guidelines for the repo [here](https://github.com/systers/powerup-android#coding-guidelines)